templates/terraform/examples/cloud_asset_folder_feed.tf.erb

```diff
@@ -17,6 +17,15 @@ resource "google_cloud_asset_folder_feed" "<%= ctx[:primary_resource_id] %>" {
     }
   }
 
+  condition {
+    expression = <<-EOT
```

Comment:
Does this not work as:
```
expression = "!temporal_asset.deleted && temporal_asset.prior_asset_state == google.cloud.asset.v1.TemporalAsset.PriorAssetState.DOES_NOT_EXIST"
```
?

Reply:
Yes, it could. But it would make the line too long. I think it can be useful to show how multiline text fields can be written in tf. Not all users may know this.